integration_tests/dataplane/mymac/mymac_test.go

```diff
@@ -124,6 +124,16 @@ func configureDUT(t testing.TB, conn *grpc.ClientConn, dut *ondatra.DUTDevice) {
 	if err != nil {
 		t.Fatal(err)
 	}
+	// Add this port to default VLAN.
+	vc := saipb.NewVlanClient(conn)
+	ctx := context.Background()
+	if _, err := vc.CreateVlanMember(ctx, &saipb.CreateVlanMemberRequest{
+		VlanId:          proto.Uint64(4095), // the default VLAN ID.
```

Comment:
shouldn't this be the OID of the VLAN object? not the vlan id

Reply:
Fixed